cloudfoundry-client-reactor/src/main/java/org/cloudfoundry/reactor/util/_UriVariable.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.reactor.util;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface _UriVariable {
+
+    String PLACEHOLDER_PATTERN = "{%s}";
```

Comment:
Any reason this shouldn't be psf?

Reply:
It is by default, since this is an interface.